.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Berikan saran dan ide untuk produk JDS Design System
+title: "[Feature Request]"
+labels: ''
+assignees: ''
+
+---
+
+**Apakah permintaan fitur berawal dari kesulitan penggunaan library? Coba jelaskan.**
+Jelaskan secara jelas dan singkat apa kesulitan yang kamu hadapi ketika menggunakan *library* **JDS Design System**. Contoh. Saya selalu frustasi ketika ingin melakukan [...].
+
+**Jelaskan solusi yang anda inginkan**
+Jelaskan secara jelas dan deskripsikan secara singkat solusi apa yang kamu inginkan. Contoh. Saya ingin komponen calendar JDS Design System bisa sama prilakunya dengan komponen calendar Carbon Design System.
```

Comment:
**super minor:**

typo wording `prilaku` should be `perilaku`

Reply:
i have fixed at https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/97/commits/86f398e121e1ef4fb6d8b118f49a056982ff9c0d